cmd/instance_show.go

```diff
@@ -108,6 +109,11 @@ func (c *instanceShowCmd) cmdRun(cmd *cobra.Command, _ []string) error {
 		Zone:            c.Zone,
 	}
 
+	out.PrivateInstance = "No"
+	if *instance.PublicIPAssignment == "none" {
```

Comment:
Let us be more robust with pointer handling:
```
if instance.PublicIPAssignment != nil && *instance.PublicIPAssignment == "none" {
```

Reply:
updated at #[e749e30](https://github.com/exoscale/cli/pull/483/commits/e749e30945b82607a5403bff065b468d27254fc3)